cob/cob.c

```diff
@@ -1945,12 +2202,16 @@ static void cob_table_tx_credit(struct m0_be_btree *tree,
 			.s_rec = m0_cob_max_nskey_size(),
 				     /* XXX ^^^^^ is it right? */
 		},
+		[COB_KVTYPE_BC] = {
+			.s_key = m0_cob_max_bckey_size(),
```

Comment:
We can change to to call the normal size functions, instead of max size.

Reply:
Done.